src/v3/src/util/idxUtils.ts

```diff
@@ -462,6 +468,33 @@ export const isValidPhoneMethodType = (
   typeof methodType !== 'undefined' && (methodType === 'sms' || methodType === 'voice')
 );
 
-export const getIDVDisplayName = (
+export const getIDVDisplayInfo = (
   transaction: IdxTransaction,
-): string | undefined => transaction.nextStep?.idp?.name;
+): IdvDisplayInfo => {
+  const idpName = transaction.nextStep?.idp?.name;
+  let termsOfUse;
+  let privacyPolicy;
+  switch (idpName?.toUpperCase()) {
+    case 'PERSONA':
+      termsOfUse = 'https://withpersona.com/legal/terms-of-use';
+      privacyPolicy = 'https://withpersona.com/legal/privacy-policy';
+      break;
+    case 'CLEAR':
+      termsOfUse = 'https://www.clearme.com/member-terms';
+      privacyPolicy = 'https://www.clearme.com/privacy-policy';
+      break;
+    case 'INCODE':
+      termsOfUse = 'https://incode.id/terms';
+      privacyPolicy = 'https://incode.id/privacy';
+      break;
+    default:
+      termsOfUse = '';
+      privacyPolicy = '';
+      break;
+  }
```

Comment:
nit: From an accessibility POV, I believe having a `default` case with `termsOfUse = undefined` and `privacyPolicy = undefined` is better than empty strings. Empty strings will make the links focusable for screenreaders, but navigate the user to a nonsensical destination, while `undefined` will prevent the links from being focusable and reduce confusion.

Reply:
Technically, `termsOfUse` and `privacyPolicy` are already `undefined` since they start out uninitialized, but it would be nice to explicitly set them to `undefined` here. A complete solution to this should probably also remove the `inline-link` class from the `<a/>` elements, but I'll leave that up to you on whether it's worth the LOE for something that really should never happen